modern/src/DevicesList.js

```diff
@@ -26,14 +28,51 @@ const useStyles = makeStyles(() => ({
     height: '25px',
     filter: 'brightness(0) invert(1)',
   },
+  batteryText: {
+    fontSize: '0.75rem',
+    fontWeight: 'normal',
+    lineHeight: '0.875rem',
+  },
+  green: {
+    color: theme.palette.common.green,
+  },
+  red: {
+    color: theme.palette.common.red,
+  },
+  gray: {
+    color: theme.palette.common.gray,
+  },
 }));
 
+const getOnlineStatus = (status) => {
```

Comment:
Maybe rename to `getStatusColor` or something like that.

Reply:
Done.